src/Framework/TestRunner/SeparateProcessTestRunner.php

```diff
@@ -252,22 +253,28 @@ static function (int $errno, string $errstr, string $errfile, int $errline): nev
         }
 
         if ($childResult !== false) {
-            if (!empty($childResult['output'])) {
-                $output = $childResult['output'];
+            if (!is_array($childResult)) {
+                $childResult = [$childResult];
             }
 
-            Facade::instance()->forward($childResult['events']);
-            PassedTests::instance()->import($childResult['passedTests']);
+            foreach ($childResult as $result) {
```

Comment:
Could this array to object transformation be extracted into a separate refactoring PR, without the actual bugfix?

Reply:
Hmm, will try. Have to close this pull request as I cannot fork again.

Reply:
You can create a new branch on your fork